src/utility/DollarPrice.tsx

```diff
@@ -0,0 +1,16 @@
+import { useCombinedStore } from '@/store/store'
+
+export default function DollarPrice() {
```

Comment:
Name it just Price, because we do not know future currency

Reply:
fixed